app/module_test.go

```diff
@@ -0,0 +1,34 @@
+package app
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/cosmos/cosmos-sdk/types"
+	govtypes "github.com/cosmos/cosmos-sdk/x/gov/types/v1"
+	"github.com/stretchr/testify/assert"
+)
+
+// Test_newGovModule tests that the default genesis state for the gov module
+// uses the utia denominiation.
+func Test_newGovModule(t *testing.T) {
+	encCfg := encoding.MakeConfig(ModuleEncodingRegisters...)
+
+	govModule := newGovModule()
+	raw := govModule.DefaultGenesis(encCfg.Codec)
+	govGenesisState := govtypes.GenesisState{}
+
+	// HACKHACK explicitly ignore the error returned from json.Unmarshal because
+	// the error is a failure to unmarshal the string StartingProposalId as a
+	// uint which is unrelated to the test here.
+	//nolint:errcheck
+	json.Unmarshal(raw, &govGenesisState)
```

Comment:
this comment is good but also why not explicitly ignore the error and get rid of the no lint?

```suggestion
	// HACKHACK explicitly ignore the error returned from json.Unmarshal because
	// the error is a failure to unmarshal the string StartingProposalId as a
	// uint which is unrelated to the test here.
	_ = json.Unmarshal(raw, &govGenesisState)
```

Reply:
Not sure how.

<img width="513" alt="Screenshot 2023-03-13 at 6 44 03 PM" src="https://user-images.githubusercontent.com/3699047/224848982-64b13357-a81f-461f-b87d-720d928d7634.png">


<img width="666" alt="Screenshot 2023-03-13 at 6 43 48 PM" src="https://user-images.githubusercontent.com/3699047/224848996-8e5e6f7a-6e00-4f18-aac5-9c25db9eb268.png">

Reply:
we just need to get rid of the `:` in the first image since we're not declaring anything

Reply:
oof how did I miss that. Can do in a follow-up PR.